nengo/utils/tests/test_functions_piecewise.py

```diff
@@ -4,6 +4,10 @@
 from nengo.exceptions import ValidationError
 from nengo.utils.functions import piecewise
 
+"""
+note:: piecewise() function is deprecated. Use Piecewise process instead.
+"""
```

Comment:
This note isn't necessary, as it's very unlikely anyone will be touching this test file. We raise the deprecation warning in `utils/functions.py` already, which is something that will be seen when running these tests anyhow.